library/script/dcp_censusdata.py

```diff
@@ -0,0 +1,29 @@
+import json
```

Comment:
Unused imports for `json` and `requests`? (copy pasta?)

Reply:
Yup - will fix now